x-pack/plugins/uptime/server/lib/requests/get_certs.ts

```diff
@@ -144,8 +144,7 @@ export const getCerts: UMElasticsearchQueryFn<GetCertsParams, CertResult> = asyn
     params.body.query.bool.filter.push(validityFilters);
   }
 
-  // console.log(JSON.stringify(params, null, 2));
-  const result = await callES('search', params);
+  const result = await callES.search(params);
```

Comment:
Unfortunately the api for the new ES client is different than the legacy one, so all of the request functions need to be updated. It also seems that the returned type has some conflicts in the handling code, at least from what I saw.